conventions.qmd

```diff
@@ -46,3 +46,31 @@ WILDS badges are maintained in a GitHub repository at <https://github.com/getwil
 
 An example badge: [![](https://getwilds.org/badges/badges/stable.svg)](https://getwilds.org/badges/#stable)
 
+### Contributing instructions
+
+All WILDS repos should have a `CONTRIBUTING.md` file. Although this file could live in many different places in a repository, in WILDS repositories it should live in the `.github/` directory.
+
+See also [GitHub's docs on CONTRIBUTING.md](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors).
+
+### Pull request templates
+
+All WILDS repos should have one or more pull request templates. These help contributors include all the relavant information, and perform any required tasks, before submitting a pull request.
+
+See also [GitHub's docs on pull request templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository)
+
+### Issue templates
+
+All WILDS repos should have one or more issue templates. These help contributors include all the relavant information, and perform any required tasks, before submitting an issue.
+
+See also [GitHub's docs on issue templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/manually-creating-a-single-issue-template-for-your-repository)
+
+
+::: {.callout-warning}
+Question for the team: Should we have organization wide files (contributing, issues, PRs) for all WILDS repos? Or some other plan?
+:::
+
+
+### Reproducible examples
+
+We strongly encourage all repos that are R packages or projects/etc. to request a "reprex" - a reproducible example. The R package [reprex](https://www.tidyverse.org/help/#reprex) makes it easy to do.
```

Comment:
Would this be a good place to talk about [pytest](https://docs.pytest.org/en/8.0.x/) for the python side?

Reply:
hmm, if there is a similar tool to `reprex` for python that'd be good to discuss here, but pytest AFAIK is purely for running tests.

there is https://pypi.org/project/reprexlite/ and https://reprexpy.readthedocs.io/en/latest/